homeassistant/components/input_slider.py

```diff
@@ -38,6 +36,31 @@
 })
 
 
+def _cv_input_slider(cfg):
+    """Config validation helper for input slider (Voluptuous)."""
+    minimum = cfg.get(CONF_MIN)
+    maximum = cfg.get(CONF_MAX)
+    if minimum >= maximum:
+        raise vol.Invalid('Maximum ({}) is not greater than minimum ({})'
+                          .format(minimum, maximum))
+    state = cfg.get(CONF_INITIAL, minimum)
+    state = max(min(state, maximum), minimum)
```

Comment:
It will be a change in current behaviour, but in the end more consistent. Will do the change